ftwhttp/client_test.go

```diff
@@ -50,7 +50,7 @@ func TestDoRequest(t *testing.T) {
 
 func TestGetTrackedTime(t *testing.T) {
 	d := &Destination{
-		DestAddr: "httpbin.org",
+		DestAddr: "httpbingo.org",
```

Comment:
At some point we should stop depending on external deps.

Reply:
Agreed. I'm overhauling all tests and will deal with this in a future PR.